be/src/exec/pipeline/scan/olap_chunk_source.cpp

```diff
@@ -521,6 +523,60 @@ Status OlapChunkSource::_extend_schema_by_access_paths() {
         int32_t root_uid = _tablet_schema->column(static_cast<size_t>(root_column_index)).unique_id();
         column.set_extended_info(std::make_unique<ExtendedColumnInfo>(path.get(), root_uid));
 
+        // Inherit default value from parent column if exists
```

Comment:
not a good idea to put the code here.
this function `_extend_schema_by_access_paths` is intended to extend the tablet schema, but this piece of code it obviously not serve the same purpose